### DISCUSSION TITLE
Custom robot not acheiving smooth movement

### DISCUSSION THREAD

**Discussion by theo-bloesch** [NVlabs/curobo#435] [Software Q&A]
I have a problem with isaac sim and curobo. I followed the tutorial from the Nvidia Curobo website: [Configuring a New Robot - cuRobo](https://curobo.org/tutorials/1_robot_configuration.html), to configure a new robot. However, when I launch the example 

omni_python curobo/examples/isaac_sim/realsense_reacher.py --robot /home/theobloesch/Documents/xArm6Curobo/xArm6CuroboConfig.yaml --show-window

The movements of my robots are not smooth, and I am unable to adjust the depth detection settings of the RealSense camera.

![image](https://github.com/user-attachments/assets/0c4b8466-c961-478d-ba30-9cb2d1f6496e)

You can view how I set up my files here : [isaac-sim-ufactory-example/xArm6/xArm6Curobo at main · heig-vd-iAi-LaRA/isaac-sim-ufactory-example · GitHub](https://github.com/heig-vd-iAi-LaRA/isaac-sim-ufactory-example/tree/main/xArm6/xArm6Curobo)

I’m working on Ubuntu 22.04 IsaacSim version 4.1 Cuda version 11.8

Has anyone encountered these issues or knows how to resolve it?

Thank you in advance !

I have already try to :

Set the gains with the extension gains tuning but my robot don’t follow the trajectories.(I made these tuning with the gripper already attached don’t know if it’s right to do so)

**Comment by balakumar-s** [comment]
Can you share a video of the robot moving so that we can understand what's happenning?

**Reply by theo-bloesch** [reply]
Yes, Sorry

Here this the video:

[Screencast from 11-21-2024 02:24:29 PM.webm](https://github.com/user-attachments/assets/4232d544-c6fa-4df9-affd-b8b1c75591c8)

Thank you in advance, and please don’t hesitate to reach out if you need any additional information

Thank you in advance !
